src/main/java/org/hmhb/kml/KmlController.java

```diff
@@ -0,0 +1,49 @@
+package org.hmhb.kml;
+
+import javax.annotation.Nonnull;
+
+import com.codahale.metrics.annotation.Timed;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * REST endpoint to serve KML for Google Maps.
+ */
+@RestController
+public class KmlController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KmlController.class);
+
+    private final KmlService service;
+
+    @Autowired
+    public KmlController(
+            @Nonnull KmlService service
+    ) {
+        LOGGER.debug("constructed");
+        this.service = requireNonNull(service, "service cannot be null");
+    }
+
+    @Timed
+    @RequestMapping(
+            method = RequestMethod.GET,
+            produces = MediaType.TEXT_PLAIN_VALUE,
```

Comment:
Google Earth (not Maps) but I found this:

> When a call is made from the client (Google Earth) to the server, the server must (1) return a response code of HTTP 200 and (2) set the response's content type to text/plain or application/vnd.google-earth.kml+xml.


From https://developers.google.com/kml/documentation/kml_tut